### ISSUE TITLE
Cannot send notification, there is no email address for this Plex User

### ISSUE THREAD

**Issue by DblTapZ** [tidusjar/Ombi#912]
#### Ombi Version:
2 different versions with same issue
V 1.10.1
V 2.0.1198

#### Update Branch:

V1.10.1 - Stable
V2.0.1198 EAB

#### Operating System:

OS:	Windows 8/Windows Server 2012
System Version:	4.0.30319.42000
Branch:	Early Access Preview
Log Level:	Info
Database Location:	C:\Ombi\Ombi.sqlite
Running Directory:	C:\Ombi

#### Applicable Logs (from `/logs/` directory or the Admin page):

```

2017/01/10 13:08:29.044 Ombi.Services.Notification.NotificationEngine Info: Notifying user Jasper 
2017/01/10 13:08:29.060 Ombi.Services.Notification.NotificationEngine Info: There is no email address for this Plex user, cannot send notification 

```
Screenshot from User Management Page showing they are a Plex user. So I'm unable to even edit their email-address
http://i.imgur.com/PCSDvBo.png

Here is a log from PlexRequests V1.10.1 with the same issue
http://pastebin.com/bfTeL0Ne
Lines 314 and 315

#### Problem Description:

When a user requests a Movie/TV Show, after the item is added to the library and the Plex Cacher runs, after it detects the movie, it says that no email exists for this user and it can't send the email. I have no manually added users via Ombi interface, all my users aside from the local Ombi administrator are from Plex. Error occurs on both my Server2012R2 and my Win8.1Pro computers

#### Reproduction Steps:

Request item from shared user account, approve admin side, and add item to Plex library, and let Ombi do a scan to search for new content. Does not seem to matter if it's PR1.10.1 or Ombi EAP

**Comment by tidusjar** [comment]
Can you send me the results of this https://plex.tv/pms/friends/all?X-Plex-Token=**YOUR API KEY**
to tidusjar@gmail.com (It could contain sensitive data)

**Comment by DblTapZ** [comment]
Sure, just sent the output to your email

**Comment by tidusjar** [comment]
Looking at the results, there is no `jasper` in your friends? Did you once have them but later removed them?

**Comment by DblTapZ** [comment]
it's under the username jld**** I set up an alias in Ombi

**Comment by DblTapZ** [comment]
So it looks like it fails to send if you have an Alias set for the user

**Comment by DblTapZ** [comment]
This is an excerpt from log files before I set the alias, and it was able to send. I'm going to try another test by removing the Alias from the User Control Panel and see if it sends

`2017/01/07 23:50:08.277 PlexRequests.Services.Notification.NotificationEngine Info: Notifying user jl******* 
2017/01/07 23:50:08.277 PlexRequests.Services.Notification.NotificationEngine Info: Sending notification to: jl******* at: REPLACED@EMAIL.COM, for title: Zootopia 
2017/01/07 23:50:08.621 PlexRequests.Services.Notification.EmailMessageNotification Info: sending message to "REPLACED@EMAIL.COM" <REPLACED@EMAIL.COM> 
 from: "REPLACED@EMAIL.COM" <REPLACED@EMAIL.COM>
 Are we authenticated: True `

**Comment by DblTapZ** [comment]
Yep that looks to be the case. Removed the Alias on another one of the users I've shared with, had them request a movie, I approved, downloaded, moved to plex. But it did not seem to detect the episode was there, so it never automatically sent the email. So I manually marked it as available, and it was able to send immediately

`2017/01/11 17:38:18.420 Ombi.Services.Notification.NotificationEngine Info: Notifying user jmj******* 
2017/01/11 17:38:18.420 Ombi.Services.Notification.NotificationEngine Info: Sending notification to: jmj******* at: REPLACED@EMAIL.COM, for title: Teen Wolf 
2017/01/11 17:38:18.843 Ombi.Services.Notification.EmailMessageNotification Info: sending message to "REPLACED@EMAIL.COM" <REPLACED@EMAIL.COM> 
 from: "REPLACED@EMAIL.COM" <REPLACED@EMAIL.COM>
 Are we authenticated: True `
